images/router/haproxy/conf/haproxy-config.template

```diff
@@ -226,6 +226,28 @@ backend be_edge_http_{{$cfgIdx}}
     {{ else }}
   balance leastconn
     {{ end }}
+
+{{ if index $cfg.Annotations "router.openshift.io/haproxy.DDOS" }}
```

Comment:
Do not use uppercase in annotations.  I don't know that these annotations have anything to do with DDOS, they're just optional fields, so they should be named appropriately.

The top level flag should be named appropriately, probably `haproxy.router.openshift.io/rate-limit-connections` and the others should follow that pattern with sub names.